iceoryx_examples/iceperf/README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-!!! hint
-    Since not all IPC mechanisms are supported on all platforms this benchmark
-    runs fully on QNX and Linux.
+!!! Note
```

Comment:
Not sure if a capital letter works with MkDocs, please check it before merging this PR.

```suggestion
!!! note
```